scriptmodules/supplementary/runcommand/runcommand.sh

```diff
@@ -677,6 +690,35 @@ function choose_fb_res() {
     load_mode_defaults
 }
 
+function user_menu() {
+    local default
+    local options=()
+    local scripts=()
+    local script
+    local i=1
+    while read -r script; do
+        script="${script##*/}"
+        script="${script%.*}"
+        scripts+=("$script")
+        options+=($i "${scripts[i-1]}")
+        ((i++))
+    done < <(find "$CONFIGDIR/all/runcommand-menu" -type f -name "*.sh" | sort)
```

Comment:
suggestion, use the same find as I said in the comment above to avoid empty/non-sh files.

```
find "$CONFIGDIR/all/runcommand-menu" -type f -name "*.sh" ! -size 0 2>/dev/null
```